deployment/proxy-deployment.yaml

```diff
@@ -0,0 +1,34 @@
+# https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: proxy
+spec:
+  defaultBackend:
+    service:
+      name: ui
+      port:
+        number: 3000
+  rules:
+  - host: staging.languagedepot.org
+    http:
+      paths:
+      - path: /api
+        pathType: Prefix
+        backend:
+          service:
+            name: lexbox-api
+            port:
+              number: 80
+      - path: /traces
```

Comment:
@myieye since we've got to route traces based on the path can we hard code that somewhere in the frontend code? I can't find it anywhere so I assume the default is `/traces` but prefer if it was explicit since we're depending on it here.